### PR TITLE
fixing most of pycodestyle E271 in pyx files

### PR DESCRIPTION
**Pull request by fchapoton**
### 📚 Description

This is fixing most of pycodestyle E271 in pyx files.

There remains a bunch in rings/polynomials.

### 📝 Checklist

- [x] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [ ] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/rings/fraction_field_element.pyx [hunk 1]
```diff
@@ -310,10 +310,10 @@ cdef class FractionFieldElement(FieldElement):
         a = self.numerator()
         b = self.denominator()
         if not root:
-            return  (a*b).is_square( root = False )
-        is_sqr, sq_rt = (a*b).is_square( root = True )
+            return (a * b).is_square(root=False)
+        is_sqr, sq_rt = ( a *b).is_square(root=True)
```
Comment (dcoudert) [inline src/sage/rings/fraction_field_element.pyx]: missing space `(a * b)`

File: src/sage/rings/fraction_field_element.pyx [hunk 2]
```diff
@@ -310,10 +310,10 @@ cdef class FractionFieldElement(FieldElement):
...
         if is_sqr:
```
Comment (mkoeppe) [inline src/sage/rings/fraction_field_element.pyx:315]: whitespace isn't right here

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by vbraun** [comment]
Merge conflict

**Comment by fchapoton** [comment]
setting back to positive after easy rebasing

**Pull request merged**